doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,72 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt?
```

Comment:
Let's mention that it's for starting a project new Level 3, 4, 5 because before writing the prompt, creating the project is the first user behavior.